articles/virtual-machine-scale-sets/virtual-machine-scale-sets-health-extension.md

```diff
@@ -13,7 +13,7 @@ ms.custom: mimckitt, devx-track-azurepowershell
 ---
 # Using Application Health extension with virtual machine scale sets
 
-Monitoring your application health is an important signal for managing and upgrading your deployment. Azure virtual machine scale sets provide support for [rolling upgrades](virtual-machine-scale-sets-upgrade-scale-set.md#how-to-bring-vms-up-to-date-with-the-latest-scale-set-model) including [automatic OS-image upgrades](virtual-machine-scale-sets-automatic-upgrade.md), which rely on health monitoring of the individual instances to upgrade your deployment. You can also use health extension to monitor the application health of each instance in your scale set and perform instance repairs using [automatic instance repairs](virtual-machine-scale-sets-automatic-instance-repairs.md).
+Monitoring your application health is an important signal for managing and upgrading your deployment. Azure virtual machine scale sets provide support for [rolling upgrades](virtual-machine-scale-sets-upgrade-scale-set.md#how-to-bring-vms-up-to-date-with-the-latest-scale-set-model) including [automatic OS-image upgrades](virtual-machine-scale-sets-automatic-upgrade.md) and [automatic vm guest patching](https://learn.microsoft.com/en-us/azure/virtual-machines/automatic-vm-guest-patching), which rely on health monitoring of the individual instances to upgrade your deployment. You can also use health extension to monitor the application health of each instance in your scale set and perform instance repairs using [automatic instance repairs](virtual-machine-scale-sets-automatic-instance-repairs.md).
```

Comment:
May want to check capitalization on key product/service names. Like Automatic VM Guest Patching over automatic vm guest patching.

Similar capitalization and/or naming patterns:
Automatic OS Image Upgrades
virtual machine scale sets
Application Health extension -- in the context of "You can also use health extension to "

Reply:
Pending this change for approval.

Reply:
Agreed, please update this for approval.